src/Appwrite/Specification/Format/Swagger2.php

```diff
@@ -286,13 +286,26 @@ public function parse(): array
                     $validator = $validator->getValidator();
                 }
 
-                $validatorClass = (!empty($validator)) ? \get_class($validator) : '';
-                if ($validatorClass === 'Utopia\Validator\AnyOf') {
+                $class = !empty($validator)
+                    ? \get_class($validator)
+                    : '';
+
+                $base = !empty($class)
+                    ? \get_parent_class($class)
+                    : '';
+
+                switch ($base) {
+                    case 'Appwrite\Utopia\Database\Validator\Queries\Base':
+                        $class = $base;
+                        break;
+                }
```

Comment:
can you run docker compose exec appwrite specs for both the latest and 1.6.x versions so we can see if this change has affected the specs in any way?

Reply:
I see $createdAt $updatedAt were added , that is expected since I added them for attributes & collections?